aws/data_source_aws_lambda_function.go

```diff
@@ -40,6 +40,22 @@ func dataSourceAwsLambdaFunction() *schema.Resource {
 					},
 				},
 			},
+			"file_system_config": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
```

Comment:
Please add `Computed: true,` as this attribute is "output-only".

Reply:
Updated.